Project_1/Styles.css

```diff
@@ -1,76 +1,150 @@
-img {
-    src: "Octopus.jpg";
-    max-width: 200%;
-    height: 700px;
-    width: 110%;
-    float: left;
+/* ==========
+GENERAL
+=============*/
+
+*:before,
+*:after {
+    Box-sizing: Border-box;
 }
 
 body {
     background-color: whitesmoke;
 }
 
+img {
+    max-width: 100%;
+}
+
+/* ===========
+TEXT STYLES 
+==============*/
+
+header {
+    border-bottom: 10px solid #018a80;
+    border-top: 10px solid #018a80;
+    margin: 50px 0 100px;
+}
+
 h1 {
-    font-size: 100px;
-    color: #d07f7d;
-    margin-right: 300px;
-    margin-left: 650px;
-    line-height: 1.3;
+    color: #a72323;
+    font-family: 'Noto Sans', sans-serif;
+    font-size: 95px;
+    margin: 10px 300px 50px 650px;
```

Comment:
I totally get what you're trying to achieve with these margins, both on this element and throughout your styles. This is a case where adding in a handful of `div`s may actually save you writing a lot of extra code; see this example:

https://codepen.io/angeliquejw/pen/d95b6e06ff67c30172a2e77d716b11ee?editors=1100

Here, I've added 2 `div`s with a `container` class, commented out a bunch of your margin declarations in the CSS and then only a bit of CSS to get close to the same effect (scroll down in the CSS to line 154 to see the CSS I added; I isolated it like this so you wouldn't hunt around for my changes. Also, I didn't tackle the author photo bit because we're going to learn a great way to achieve that layout in class this week. 👍🏻